src/test/java/com/wikia/webdriver/testcases/mercurytests/LoginTests.java

```diff
@@ -73,12 +73,11 @@ public void anonCanNotLogInWithInvalidUsername() {
             .getTopbar()
             .openNavigation()
             .clickOnSignInRegisterButton()
-            .clickSignInButton()
-            .getLoginArea()
-            .typeUsername(String.valueOf(DateTime.now().getMillis()))
-            .typePassword(Configuration.getCredentials().password10)
-            .clickSignInButtonToGetError()
-            .verifyErrorMessage(expectedErrorMessage);
+            .navigateToSignIn();
+            //.typeUsername(String.valueOf(DateTime.now().getMillis()))
+            //.typePassword(Configuration.getCredentials().password10)
+            //.clickSignInButton()
+            //.verifyErrorMessage(expectedErrorMessage);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3ACommentedOutCodeLine)